children/tests/test_api.py

```diff
@@ -41,6 +46,21 @@
 from ..models import Child, Relationship
 
 
+@pytest.fixture(
+    params=[
+        field.name
+        for field in Child._meta.get_fields()
+        if field.name not in ["id", "notes"]
+    ]
```

Comment:
Trivial note: the list used as a parameter could be initialized in a variable so it would be more clear what they are.

Reply:
Taken into account in 93256b48babb5e17bc8336791d7b77e475471f2a